docs/UserGuide.md

```diff
@@ -40,147 +30,332 @@ By : `Team SE-EDU`  &nbsp;&nbsp;&nbsp;&nbsp; Since: `Jun 2016`  &nbsp;&nbsp;&nbs
 > * Items in `SQUARE_BRACKETS` are optional.
 > * Items with `...` after them can have multiple instances.
 > * Parameters can be in any order.
+> * Options with '/' allow either word to be used.
+
+### 2.0. Viewing tasks
+Viewing all tasks in the todo list<br>
+Format: `list`
+> * A list of tasks will always be displayed.
+> * When starting the program, the list will show all tasks which are currently not completed.
+> * When performing find operations, this list will be updated to show only the results searched for.
 
 ### 2.1. Viewing help : `help`
 
 Format: `help`
 
-> Help is also shown if you enter an incorrect command e.g. `abcd`
+> Help is also shown if you enter an incorrect command. e.g. `abcd`
 
-### 2.2. Adding a person: `add`
+### 2.2. Adding a task: `add`
 
-Adds a person to the address book<br>
-Format: `add NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...`
+Adds a task to the todo list<br>
+Format: `add NAME [startdate/STARTDATE] [enddate/ENDDATE] [recurring/PERIOD(day/week/month)] [priority/PRIORITY] [tag/TAG]...`
 
-> Persons can have any number of tags (including 0)
+> Tasks can have any number of tags (including 0).
 
 Examples:
 
-* `add John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01`
-* `add Betsy Crowe t/friend e/betsycrowe@gmail.com a/Newgate Prison p/1234567 t/criminal`
+* `add Do Homework`
+* `add Meeting With Boss enddate/11-11-2011 19:30 startdate/11-11-2011 17:30`
+* `add Check Email startdate/today recurring/day`
 
```

Comment:
How about some more examples of the different types of tasks your application supports?
Eg. Floating tasks, deadline tasks.